### PR TITLE
VACMS-9330: Update template to include fallback for non-video media e…

### LINKED ISSUE #9330: Embedded media template updates
## Describe the defect
When using embedded images in KB articles, the image does not display. 

## To Reproduce
1. Go to https://prod.cms.va.gov/help/vamc/about-locations-content-for-vamcs/about-alerts-and-operating-statuses/best-practices-for-vamc-system-banner-alerts
2. Scroll down to section 'Use links'
3. See that there's no embedded image preceding that section

## Expected behavior
Expect an embedded image to appear 

## Additional context
Add any other context about the problem here. Reach out to the Product Managers to determine if it should be escalated as critical (prevents users from accomplishing their work with no known workaround and needs to be addressed within 2 business days).

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by msbtterswrth**
…mbeds.

## Description

closes #9330 

## Testing done
manual

## Screenshots
image

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As a CMS user
1. View a KB article page with an embedded image (`help/vamc/about-locations-content-for-vamcs/about-alerts-and-operating-statuses/best-practices-for-vamc-system-banner-alerts`)
   - [x] Validate that an image is visible on the page
2. Edit a KB article and add an embedded video (`help/vamc/about-locations-content-for-vamcs/about-alerts-and-operating-statuses/how-to-edit-covid-19-status-for-a-vamc-facility`)
   - [x] Validate the video title/duration appear above the video

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by rachel-kauff — APPROVED** [review]

**Comment by swirtSJW** [comment]
QA steps seem incomplete.  Changes were made to video display but there is no step to QA that.

**Comment by msbtterswrth** [comment]
We threaded out moving the title/duration above the video [here](https://dsva.slack.com/archives/CT4GZBM8F/p1654532515841519?thread_ts=1654531279.460899&cid=CT4GZBM8F). I missed updating the QA steps, but that has been fixed now.

**Review by swirtSJW — DISMISSED** [review]
QA passes.  

Adding the title above the video is good.  However, there is still nothing that tells me this is a video.
image
I get a heading read to me, a time, a share link, then play button.

That's why I recommended this format
image


Approving it because it is better than what we have.

File: docroot/themes/custom/vagovclaro/templates/misc/media--embedded.html.twig [hunk 1]
```diff
@@ -23,10 +23,14 @@
 <article{{ attributes.addClass(classes) }}>
   {{ title_suffix.contextual_links }}
   {% if content %}
-    {{content.field_media_video_embed_field}}
-    <h3>{{content.name.0}}</h3>
-    {% if content.field_duration.0 %}
-      <p><strong>Duration:</strong> {{content.field_duration.0}}</p>
+    {% if content.field_media_video_embed_field %}
+      <h3>{{content.name.0}}</h3>
```
Comment (swirtSJW) [inline docroot/themes/custom/vagovclaro/templates/misc/media--embedded.html.twig:27]:
```suggestion
      <h3>Video: {{content.name.0}}</h3>
```

**Review by swirtSJW — APPROVED** [review]

**Pull request merged**